iz_helpers/run.py

```diff
@@ -149,7 +149,8 @@ def create_zoom_single(
             width,
             height,
         )
-        current_image = processed.images[0]
+        if(len(processed.images) > 0):
```

Comment:
what we do if no image ? could be exception?!
same for all occurances of "if len(processed)"...

Reply:
I tested it out extensively and Have yet to get it to fail. 
![image](https://user-images.githubusercontent.com/5393103/233874753-37cf8861-dad5-4931-800f-a3bab7d3e8fe.png)

This is what it does right now if it's at zero and the web UI works as well.
![image](https://user-images.githubusercontent.com/5393103/233874798-f9e53072-aeb2-4353-b1ba-55bcaa0cdbdd.png)

Reply:
@GeorgLegato this is mainly to fix what happened when you clicked interrupt on the web UI. I asked @v8hid if I could fix the problem with my patch and he gave the go-ahead. I can post the discord snippet if needed.

Reply:
> what we do if no image ? could be exception?! same for all occurances of "if len(processed)"...

Also, why would we need this as an exception? This only happens when the user clicks interrupt. Right now I just having it return the black frame that was created at the start of the function.